handlers/syslog/syslog.go

```diff
@@ -0,0 +1,68 @@
+// Package syslog implements output to local or remote hosts via the syslog protocol.
+package syslog
+
+import (
+  "fmt"
+  "log/syslog"
+
+  "github.com/apex/log"
+)
+
+// Handler implementation.
+type Handler struct {
+  log *syslog.Writer
+  // map apex levels to syslog levels.
+  levelsMap map[log.Level](func (m string) error)
+}
+
+// New syslog handler.
+//  addr: network address of syslog host. (default: localhost)type syslogOutput func (w *syslog.Writer)
```

Comment:
`type syslogOutput func (w *syslog.Writer)` looks like it shouldn't be here :D accidental paste or something